app/models/journeys/page_sequence.rb

```diff
@@ -68,7 +68,7 @@ def incomplete_slugs
     def can_skip_next_slug?
       # This allows 'address' page to be skipped when the postcode is present
       # Occurs when populated from 'postcode-search' and the subsequent 'select-home-address' screens
-      true if current_slug == "select-home-address" && claim.postcode.present?
+      true if ["select-home-address", "postcode-search"].include?(current_slug) && claim.postcode.present?
```

Comment:
There is an option for the user to enter their address manually rather than using the psotcode search - this effectively alters the page sequence because the 'select address' page is skipped.